jena-arq/src/main/java/org/apache/jena/riot/system/RiotLib.java

```diff
@@ -52,160 +38,199 @@
 
-    /** Create a parser profile for the given setup */
+    /**
+     * Create a parser profile for the given setup
+     */
     private static ParserProfile profile(String baseIRI, boolean resolveIRIs, boolean checking, ErrorHandler handler) {
         LabelToNode labelToNode = SyntaxLabels.createLabelToNode();
         IRIx base = resolveIRIs
                 ? IRIs.resolveIRI(baseIRI)
                 : IRIx.create(baseIRI);
-        IRIxResolver resolver = IRIxResolver.create(base).resolve(resolveIRIs).allowRelative(false).build();
-        ParserProfile profile = RiotLib.createParserProfile(factoryRDF(labelToNode), handler, resolver, checking);
-        return profile;
+        IRIxResolver resolver = IRIxResolver
+                .create(base)
+                .resolve(resolveIRIs)
+                .allowRelative(false)
+                .build();
+        return RiotLib.createParserProfile(factoryRDF(labelToNode), handler, resolver, checking);
     }
 
-    /** Create a new (not influenced by anything else) {@code FactoryRDF}
+    /**
+     * Create a new (not influenced by anything else) {@code FactoryRDF}
      * using the label to blank node scheme provided.
      */
     public static FactoryRDF factoryRDF(LabelToNode labelMapping) {
         return new FactoryRDFCaching(FactoryRDFCaching.DftNodeCacheSize, labelMapping);
     }
 
-    /** Create a new (not influenced by anything else) {@code FactoryRDF}
+    /**
+     * Create a new (not influenced by anything else) {@code FactoryRDF}
      * using the default label to blank node scheme.
      */
     public static FactoryRDF factoryRDF() {
         return factoryRDF(SyntaxLabels.createLabelToNode());
     }
 
-    /** Create a {@link ParserProfile} with default settings. */
+    /**
+     * Create a {@link ParserProfile} with default settings.
+     */
     public static ParserProfile dftProfile() {
         return createParserProfile(RiotLib.factoryRDF(), ErrorHandlerFactory.errorHandlerStd, true);
     }
 
-    /** Create a {@link ParserProfile} with default settings, and a specific error handler. */
-    public static ParserProfile createParserProfile(ErrorHandler errorHandler) {
-        return createParserProfile(RiotLib.factoryRDF(), errorHandler, true);
-    }
-
-    /** Create a {@link ParserProfile} with default settings, and a specific error handler. */
+    /**
+     * Create a {@link ParserProfile} with default settings, and a specific error handler.
+     */
     public static ParserProfile createParserProfile(FactoryRDF factory, ErrorHandler errorHandler, boolean checking) {
         return new ParserProfileStd(factory,
-                                    errorHandler,
-                                    IRIxResolver.create(IRIs.getSystemBase()).build(),
-                                    PrefixMapFactory.create(),
-                                    RIOT.getContext().copy(),
-                                    checking, false);
+                errorHandler,
```

Comment:
In this case, I see your point. In a more general situation, I like to prevent scrolling to the right by breaking lines. But that's personal as well ;-)